frontend/fingertips-frontend/.gitignore

```diff
@@ -12,6 +12,7 @@
 
 # testing
 /coverage
+*.png
```

Comment:
do we want to gitignore this?

Reply:
ah meant to remove this on last PR but then closed it and forgot - will remove it ta